pipelines/optimus/Optimus.changelog.md

```diff
@@ -1,16 +1,17 @@
 # optimus_v3.0.0
 2020-06-10 (Date of Last Commit)
 
-* Removed zarr formatted matrix and metrics outputs and replaced with Loom
-* Removed emptyDrops for sn_rna mode
-* Updated Loom file attribute names: CellID to cell_names, Gene to gene_names, and Accession to ensembl_ids
+* Removed the Zarr formatted matrix and metrics outputs and replaced with Loom
+* Removed EmptyDrops for sn_rna mode
+* Updated the Loom file attribute names: CellID to cell_names, Gene to gene_names, and Accession to ensembl_ids
 * Added metrics for mitochondrial reads
+* Changed the name of the merged.bam output to sample_id.bam
```

Comment:
If we can update the pipeline to accept an optional input I think this should say 

“Changed the default bam output name from ‘merged.bam’ to the input ‘<sample_id>.bam’. Added an optional input ‘bam_output_basename’ to choose your own output name” 

Or something like that.